### PR TITLE
Add language server 2201.2.2 release note 

### PR DESCRIPTION
**Pull request by gayalkuruppu**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md [hunk 1]
```diff
@@ -40,6 +40,26 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.2.2](https://git
 
 ## Developer tools updates
 
+### New features
+
+#### Language Server
+
+- Implemented a new code action to extract a constant expression to a constant
+- Added a new code action to convert an array to array mapping into a query expression
+- Introduced new API to load a project into the Workspace Manager
```
Comment (IMS94) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:49]: @gayalkuruppu we added following new features as well:
1. Extract to function
2. Extract to local variable
3. Rename popup for create variable code action <- Let's add this under a new feature

File: test-scripts/ballerina-test-automation/.settings/org.eclipse.buildship.core.prefs [hunk 2]
```diff
@@ -1,2 +1,13 @@
+arguments=--init-script /var/folders/sb/7j7ln5gn72g1sc15bm9l2vgm0000gn/T/init15326630906069179233.gradle --init-script /var/folders/sb/7j7ln5gn72g1sc15bm9l2vgm0000gn/T/init10568042303374396679.gradle
```
Comment (IMS94) [inline test-scripts/ballerina-test-automation/.settings/org.eclipse.buildship.core.prefs:1]: @gayalkuruppu these are non-related changes right?

**Review by IMS94 — CHANGES_REQUESTED** [review]

File: test-scripts/ballerina-test-automation/.settings/org.eclipse.buildship.core.prefs (hunk 2 above)
Comment (gayalkuruppu) [reply test-scripts/ballerina-test-automation/.settings/org.eclipse.buildship.core.prefs:1]: Yeah will remove them

**Review by mohanvive — APPROVED** [review]

**Comment by praneesha** [comment]
@gayalkuruppu - Please send these updates to https://github.com/ballerina-platform/ballerina-dev-website/tree/dev/downloads/swan-lake-release-notes/swan-lake-2201.2.2 as the release note is already live now.

**Comment by IMS94** [comment]
Sent in https://github.com/ballerina-platform/ballerina-dev-website/pull/5685. Hence closing

**Pull request closed without merging**